registry/storage/driver/gcs/gcs.go

```diff
@@ -247,14 +267,8 @@ func (d *driver) Name() string {
 // GetContent retrieves the content stored at "path" as a []byte.
 // This should primarily be used for small objects.
 func (d *driver) GetContent(context context.Context, path string) ([]byte, error) {
```

Comment:
As an aside, I'd like the name of this func arg to be `ctx` -- just for consistency sake and it's less typing. We need to do it across the whole of this file actually.